### PR TITLE
platform: version info

### PR DESCRIPTION
**Pull request by LouManglass**
## Why
Currently there's no insight into how well we support various platforms and our response is very reactive. It's also difficult to assess the scope of impact of platform-specific issues. More insight allows us to better prioritize response and be proactive to environmental patterns.

## What
platform, version, arch are now noted.

## Validation
* [ ] PR Rules pass (CI passes, review approved)
* [x] Manually tested by me

### CODE REVIEW

File: workspaces/ui/src/utilities/useAnalyticsHook.js [hunk 1]
```diff
@@ -72,6 +73,9 @@ export function AnalyticsContextStore({ children, specService }) {
       const clientContext = {
         clientAgent: clientAgent,
         clientId: clientId,
+        //platform: os.platform(),
```
Comment (devdoshi) [inline workspaces/ui/src/utilities/useAnalyticsHook.js:76]: Should there be any changes to this file?

File: workspaces/cli-server/src/analytics.ts [hunk 2]
```diff
@@ -19,6 +19,9 @@ import {getOrCreateAnonId} from "@useoptic/cli-config/build/opticrc/optic-rc";
 const packageJson = require('../package.json');
 
 const clientId = `local_cli_${packageJson.version}`;
+const platform = os.platform();
```
Comment (devdoshi) [inline workspaces/cli-server/src/analytics.ts:22]: Looks good. See also: `workspaces/cli-scripts/src/errors.ts` for what we send to Sentry

File: workspaces/ui/src/utilities/useAnalyticsHook.js (hunk 1 above)
Comment (LouManglass) [reply workspaces/ui/src/utilities/useAnalyticsHook.js:76]: Thanks, good catch. No. Reverted in upcoming push.

File: workspaces/cli-server/src/analytics.ts (hunk 2 above)
Comment (LouManglass) [reply workspaces/cli-server/src/analytics.ts:22]: Thanks, it appears that Sentry already grabs some of this, so I added the host's release information as well.

File: workspaces/cli-scripts/src/errors.ts [hunk 3]
```diff
@@ -29,11 +29,13 @@ export function trackWithSentry({
   const hostPlatform = process.platform;
   const hostCpuCount = Os.cpus().length.toString();
   const hostMemorySize = Os.totalmem().toString();
+  const hostRelease = Os.release();
```
Comment (devdoshi) [inline workspaces/cli-scripts/src/errors.ts:32]: I forgot we've duplicated this code in `cli-server/src/errors.ts` as well - would you mind applying it there too?

**Review by devdoshi — APPROVED** [review]
Looks good - just one small comment left

**Comment by devdoshi** [comment]
clear to merge!

File: workspaces/cli-scripts/src/errors.ts (hunk 3 above)
Comment (LouManglass) [reply workspaces/cli-scripts/src/errors.ts:32]: Looks like that's already done, awesome!

**Pull request merged**